supply_land.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: index
+title: Supply of Food, Feed, and Forestry
+prev: inputs_supply.html
+next: outputs_quantity.html
+gcam-version: v5.3 
+---
+
+# Table of Contents
+
+- [Inputs to the Model](#inputs-to-the-model)
+- [Description](#description)
+- [Equations](#equations)
+- [Insights and intuition](#insights-and-intuition)
+- [Policy options](#policy-options)
+- [IAMC Reference Card](#iamc-reference-card)
+- [References](#references)
+
+## Inputs to the Model
+**Table 1: Inputs required by the supply model <sup>[1](#table_footnote)</sup>**
```

Comment:
Table might be easy to partially complete by copying over entries from inputs_supply and inputs_land files